### PR TITLE
Adding slurmpter

### PR DESCRIPTION
**Pull request by isaac-cfwong**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/slurmpter/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a4dd1eeb5692ce2e92847fb01e73430323c912c3132d53418f0bd1f0ea5dcd34
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
```
Comment (synapticarbors) [inline recipes/slurmpter/meta.yaml:17]: Under `host` you only need `pip` and `python`. The other packages do not appear to be used directly by `setup.py` or indirectly due to an import in that file. 

Also under both `host` and `run` changes `python` to `python >=3.5`

File: recipes/slurmpter/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a4dd1eeb5692ce2e92847fb01e73430323c912c3132d53418f0bd1f0ea5dcd34
+
+build:
```
Comment (synapticarbors) [inline recipes/slurmpter/meta.yaml:12]: It looks like this package can be `noarch: python`. See:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

File: recipes/slurmpter/meta.yaml [hunk 3]
```diff
@@ -17,11 +17,11 @@ build:
 requirements:
   host:
     - pip
-    - python
+    - python>=3.5
```
Comment (synapticarbors) [inline recipes/slurmpter/meta.yaml:20]: You need a space between i.e `python >3.5` both here and under `run`

File: recipes/slurmpter/meta.yaml (hunk 1 above)
Comment (isaac-cfwong) [reply recipes/slurmpter/meta.yaml:18]: Fixed. Thanks.

File: recipes/slurmpter/meta.yaml (hunk 2 above)
Comment (isaac-cfwong) [reply recipes/slurmpter/meta.yaml:12]: Yes the package is pure Python. I've added noarch to it.

File: recipes/slurmpter/meta.yaml (hunk 3 above)
Comment (isaac-cfwong) [reply recipes/slurmpter/meta.yaml]: FIxed.

**Comment by synapticarbors** [comment]
Just to be clear, you want both `noarch: python` as well as `python >=3.5` under `host` and `run`. The latter is because of the `python_requires` in the `setup.py`. It is fine to have the constraint and the `noarch`.

**Comment by isaac-cfwong** [comment]
> Just to be clear, you want both `noarch: python` as well as `python >=3.5` under `host` and `run`. The latter is because of the `python_requires` in the `setup.py`. It is fine to have the constraint and the `noarch`.

I see. I saw the automatic message "noarch: python recipes are recommended to have a lower bound on minimum python bound. This recommendation will become requirement in a future version." above. I thought the lower bound has to be specified right after "noarch: python", but seemingly I misunderstood it.

**Comment by isaac-cfwong** [comment]
The osx_osx_64 job was in queue for a long time without response. I added two more not very important import tests to re-init the pipeline. Is there any other way to restart the pipeline without pushing some irrelevant commits?

**Comment by synapticarbors** [comment]
@isaac-cfwong - I am able to restart the CI along with anyone else on the core team. Just ping us if you need assistance.

**Comment by isaac-cfwong** [comment]
> @isaac-cfwong - I am able to restart the CI along with anyone else on the core team. Just ping us if you need assistance.

Ok. Thanks.

**Comment by synapticarbors** [comment]
@isaac-cfwong -- Thank you for your contribution to conda-forge! If you run into any issues once the feedstock for this recipe has been created, please don't hesitate to reach out for help.

**Pull request merged**